QModManager/DataStructures/SortedCollection.cs

```diff
@@ -139,7 +139,7 @@ private void LinkAfterPreferences(List<IdType> orderedPreferences, List<SortedTr
                     {
                         if (!item.IsLinked)
                         {
-                            node.SetBefore(item);
+                            node.SetAfter(item);
```

Comment:
Idk, maybe it's just me, but when I see such a line I read it like this: set `node` after `item` (same as I read `node.SetNodeAfter(item)`) and not like set some `After` variable for `node` to `item`.

Sorting is now works as intended btw.

Reply:
Pushed some renaming updates that should make things a little more readable